lizzy/models/stack.py

```diff
@@ -81,14 +84,36 @@ def generate_id(self) -> str:
         """
         return '{name}-{version}'.format(name=self.stack_name, version=self.stack_version)
 
-    def _parse_date(self, date_time):
-        if isinstance(date_time, datetime):
-            return date_time
-        return date_parser.parse(date_time)
-
     @classmethod
     def get(cls, *args, **kwargs) -> "Stack":
         try:
             return super().get(*args, **kwargs)
         except KeyError:
             raise ObjectNotFound(args[0])
+
+    @property
+    def cf_stack(self) -> Dict[str, Any]:
+        """
+        Gets the Cloud Formation Stack
+        """
+
+        if self.__cf_stack is None:
+            cloud_formation = boto3.client('cloudformation')
+            try:
+                stacks = cloud_formation.describe_stacks(StackName=self.stack_id)  # type: Dict[str, Any]
+            except botocore.exceptions.ClientError:
+                # don't save the {} because it can work later
```

Comment:
What is the context of this comment? It is generating more confusion than clarity. Maybe remove or rewrite it in a clear way?